sample/Cloud/README.md

```diff
@@ -0,0 +1,132 @@
+# OpenTok Cloud API Sample Python
+
+This is a simple demo app that shows how you can use the OpenTok-Python-SDK to register
+callbacks to receive notifications for streams and connections created and destroyed with your
+API Key and also to send arbitrary messages and disconnect participants in OpenTok sessions
+from your server.
+
+## Running the App
+
+First, we highly recommend setting up a [virtualenv](http://www.virtualenv.org/en/latest/).
+
+```
+$ virtualenv venv
+$ source venv/bin/activate
+```
+
+Next, download the dependencies using [Pip](http://www.pip-installer.org/en/latest/), from the
+current directory:
+
+```
+(venv)$ pip install -r requirements.txt
+```
+
+Then add your own API Key and API Secret to the environment variables. There are a few ways to do
+this but the simplest would be to do it right in your shell.
+
+```
+(venv)$ export API_KEY=0000000
+(venv)$ export API_SECRET=abcdef1234567890abcdef01234567890abcdef
+```
+
+This sample app needs to be reachable from OpenTok servers in order to receive notifications and
+usually your machine doesn't have a public address.   There are a number of public services that
+can help you overcome that limitation.    For example you can use [localtunnel](https://localtunnel.github.io).
+
+First install localtunnel and start the process to listen for HTTP requests in the port where your
+server is listening (by default 5000 in this sample app).
+```
+npm install -g localtunnel
+lt --port 5000
+```
+
+You will get a public url from the "lt" command and you have to add it to the environment variables:
+
+```
+(venv)$ export PUBLIC_URL=http://orfsvirfmv.localtunnel.me
+```
+
```

Comment:
I used ngrok instead of localtunnel. I'm not sure, but ngrok may have more active development than localtunnel.